internal/backend/httpsvc/intercept.go

```diff
@@ -72,7 +73,11 @@ func intercept(z *zap.Logger, cfg *LoggerConfig, extractors []RequestLogExtracto
 		startTime := time.Now()
 		r = r.WithContext(context.WithValue(r.Context(), startTimeCtxKey, startTime))
 
-		next.ServeHTTP(rwi, r)
+		if strings.HasPrefix(r.URL.Path, "/autokitteh.remote.v1.Worker") {
```

Comment:
Maybe a custom mux instead? No sure.

Reply:
yup.

Reply:
refactored to inject mux using fx only in server init (fx.Invoke)
good idea
thanks